### ISSUE TITLE
Inputting MAF for Exact Sharing Probability 

### ISSUE THREAD

**Issue by klmartinez** [sherman5/RVS#7]
I am having trouble understanding how to incorporate minor allele frequency information to calculate an exact sharing probability that can then be used in the `multipleVariantPValue `function.

I see that for `RVsharing `function you can designate the `alleleFreq`, so does this need to be in the form of a data frame or a list with a MAF for every variant being analyzed? Or how would I properly incorporate this information?

**Comment by sherman5** [comment]
This type of integration isn't supported at the moment so it would require wrapping the calls to RVsharing and multipleFamilyPValue. As far as I understand, you would want to go through each variant and calculate the sharing probabilities based on a MAF for that variant, and then call multipleFamilyPValue on that variant given the newly calculated sharing probabilities?

**Comment by klmartinez** [comment]
Yes, that would be my ideal goal. I wanted to use the `multipleVariantPValue `function but to only include variants that are actually rare in the population. So maybe I will try to filter out variants based on MAF before converting them to plink files and being run through RVS.

**Comment by sherman5** [comment]
That actually brings up an important point - the multipleVariantPValue function inherently assumes all variants being passed are rare in the population. The calculation of the sharing probabilities are based on this assumption. @abureau can you confirm/clarify this?

**Comment by abureau** [comment]
Sorry for the delay, this issue was raised during my vacations. The calculation of the p-value in the multipleVariantPValue function assumes sharing probabilities contained in the sharingProbs vector are the same for all variants. You get identical sharing probabilities for all variants when you assume they are rare, i.e. the limiting sharing probability when their frequency tends to 0. Implementing the computation of multiple variant p-values when the sharing probabilities depend on the variant frequency would be difficult as it would involve cumbersome book-keeping, but is conceptually possible.
